bitnami/spark/README.md

```diff
@@ -0,0 +1,256 @@
+| `image.pullPolicy`                          | spark image pull policy                                                                     | `IfNotPresent`                                          |
+| `image.pullSecrets`                         | Specify docker-registry secret names as an array                                            | `[]` (does not add image pull secrets to deployed pods) |
+| `master.debug`                              | Specify if debug values should be set on the master                                         | `false`                                                 |
+| `worker.debug`                              | Specify if debug values should be set on workers                                            | `false`                                                 |
+| `master.webPort`                            | Specify the port where the web interface will listen on the master                          | `8080`                                                  |
+| `master.clusterPort`                        | Specify the port where the master listens to communicate with workers                       | `7077`                                                  |
+| `master.daemonMemoryLimit`                  | Set the memory limit for the master daemon                                                  | No default                                              |
+| `master.configOptions`                      | Optional configuration if the form `-Dx=y`                                                  | No default                                              |
+| `master.updateStrategy`                     | Update strategy for the deployment                                                          | `RollingUpdate`                                         |
+| `master.replicaCount`                       | Number of master replicas                                                                   | `1`                                                     |
+| `master.securityContext.enabled`            | Enable security context                                                                     | `true`                                                  |
+| `master.securityContext.fsGroup`            | Group ID for the container                                                                  | `1001`                                                  |
+| `master.securityContext.runAsUser`          | User ID for the container                                                                   | `1001`                                                  |
+| `master.nodeSelector`                       | Node labels for pod assignment                                                              | `{}`                                                    |
```

Comment:
Can you change nodeSelector, tolerations and affinity to be treated as templates?

Example: https://github.com/bitnami/charts/blob/master/upstreamed/redis/templates/redis-master-statefulset.yaml

Make sure you specify it in the README

Reply:
They are treated as templates, I will include on the README.